docs/ops/queue.toml

```diff
@@ -22,3 +22,170 @@ description = """
 Getting the value the average of the values in the queue. Setting `x` sets the
 value of each entry in the queue to `x`.
 """
+
+["Q.CLR"]
+prototype = "Q.CLR"
+prototype_set = "Q.CLR x"
+short = "Clear queue"
+description = """
+Clear queue, set all values to 0, length to 1. If parameter `x` is provided, set first elements to `x`.
```

Comment:
is it possible to have queue length of 0?

Reply:
No, mainly to keep in line with the general though of old Q.

Reply:
i can’t really comment on this as i don’t use the queue, but to me it doesn’t make sense to have a queue that always has at least 1 element. but if that’s what the original implementation does then makes sense to keep it that way.

Reply:
If you clear the queue then add an element, does the residual 0 get pushed into the next spot, or overwritten?

Reply:
It depends on GRW (grow)

Q.CLR // now Q is 1 long and all 0s
Q.CLR 42 // now Q is 1 long, first element (idx 0) is 42, the rest are 0
Q 1 // length is still 1, first elemtn is 1, 2nd element is 42, but "past the length border"
Q.GRW 1 // now queue grows automatically
Q 2 // now Q is 2 long, first 2, 2nd 1. A third element 42 is "behing the length border", the rest are 0

Yeah, I didn't dream up the org Q, took me ages to figure out (and document) how that worked, quirky as is it, I'm just trying to extend what's already there.

Not saying RTM at all, but I propose a read through the examples in the docs, I hope it's clearer after that :-)

Reply:
i'd say this is the case where community input is more valuable than any feedback i would be able to provide :)

Reply:
I agree, the original post didn't stir anything up (except a few enthusiastic comments)
https://llllllll.co/t/teletype-3-2-feature-requests-and-discussions/32052/345

Could we put it in the beta as-is and get feedback once people have tried it out themselves?

Reply:
Ok I did a re-read of the docs and it's much clearer to me now. The only thing which is not mentioned anywhere is whether the queue-modifying OPs only affect in-bound values?

Reply:
sounds like a question for @tehn 
(my personal view is - we don't want to end up with a scenario similar to patterns where start/end/length usage is confusing and inconsistent, and adding new ops will make it more difficult to change the approach if needed. but i shouldn't really comment as i don't use the queue and therefore my opinion here might be just creating noise)

Reply:
I will go over the docs again, but everything respects in-bound values. EXCEPT direct indexing Q.I and copy from pattern